### PR TITLE
Update cross character

### LINKED ISSUE #92: Main `cross` character is incorrect
(discovered while working on ava test result accessibility: https://github.com/avajs/ava/pull/3082 )

The main `cross` character is U+2716 HEAVY MULTIPLICATION X "✖︎", but for accessibility (and proper contrast with `tick` U+2714 HEAVY CHECK MARK "✔︎") it should instead be U+2718 HEAVY BALLOT X "✘︎".

### PR DESCRIPTION
**Pull request by gibson042**
Fixes #92

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ String where the Unicode symbols will be replaced with fallback symbols dependin
 | tick                                        | `✔`  |   `√`   |
 | info                                        | `ℹ`  |   `i`   |
 | warning                                     | `⚠`  |   `‼`   |
-| cross                                       | `✖`  |   `×`   |
+| cross                                       | `✘`  |   `×`   |
```
Comment (ehmicky) [inline readme.md:75]: The following test needs to be fixed:

https://github.com/sindresorhus/figures/blob/62b5c69bd1ff54a523857b89f1fad184fb553ba5/test.js#L17-L18

**Comment by ehmicky** [comment]
Thanks a lot @gibson042!

**Review by ehmicky — CHANGES_REQUESTED** [review]
Looks good! Just one small suggestion.

File: readme.md (hunk 1 above)
Comment (gibson042) [reply readme.md:75]: Thanks, updated.

**Review by ehmicky — APPROVED** [review]
This looks good! 
What do you think @sindresorhus?

**Comment by gibson042** [comment]
This is probably a minor point, but U+2716 HEAVY MULTIPLICATION X "✖︎" and U+2714 HEAVY CHECK MARK "✔︎" are both technically emoji while U+2718 HEAVY BALLOT X "✘︎" is not. Another possible alternative to the latter would be U+274C CROSS MARK "❌︎", but that one has the further complication of having Emoji_Presentation (e.g., its default form when not forced into text as I have done in the preceding clause is "❌"). That characteristic is also true of U+2705 WHITE HEAVY CHECK MARK "✅︎"/"✅", but outlined glyphs don't really work in a terminal context.

All told, I'm pretty confident that U+2718 HEAVY BALLOT X "✘︎" is the right choice, but it is still worth providing the context. And if the mismatch _is_ a problem, it is possible to suffix the emoji characters with U+FE0E VARIATION SELECTOR-15 to force text presentation as I have done here.

**Pull request merged**